Modules/mathmodule.c

```diff
@@ -3234,16 +3328,24 @@ math_prod_impl(PyObject *module, PyObject *iterable, PyObject *start)
 static PyObject *
 perm_comb_small(unsigned long long n, unsigned long long k, int iscomb)
 {
+    static const unsigned char fast_comb_limits1[] = {
+        127, 127, 127, 127, 127, 127, 127, 127,  // 0-7
+        127, 127, 127, 127, 127, 127, 127, 127,  // 8-15
+        116, 105, 97, 91, 86, 82, 78, 76,  // 16-23
+        74, 72, 71, 70, 69, 68, 68, 67,  // 24-31
+        67, 67, 67,  // 32-34
+    };
     /* long long is at least 64 bit */
-    static const unsigned long long fast_comb_limits[] = {
+    static const unsigned long long fast_comb_limits2[] = {
         0, ULLONG_MAX, 4294967296ULL, 3329022, 102570, 13467, 3612, 1449,  // 0-7
         746, 453, 308, 227, 178, 147, 125, 110,  // 8-15
```

Comment:
Looks like this table isn't needed anymore for entries beyond `fast_comb_limits2[13]`. At 14 and above, the `n` value stored is smaller than the corresponding entry in `fast_comb_limits1`, so all such cases that could pass the `fast_comb_limits2` test will be handled already by the earlier `fast_comb_limits1` test.

Reply:
Yes.